src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null) {
+            return userService.findByEmail(email).getPassword().equals(password);
+        } else {
+            return false;
+        }
```

Comment:
Please simplify this expression for one line after return that result in boolean